.gitignore

```diff
@@ -14,3 +14,4 @@ env/
 .next/
 *.log
 temp-landing-page/
+.env*.local
```

Comment:
Question: why this pattern for the environment variable? is this some standard web frameworks use?

Reply:
Correct! Whenever you run `vercel env pull` the name of the environment variables file is `.env.local`. 
There can also be variations to this name depending on our vercel setup.
So to git-ignore it, you must much use such patterns.